.github/workflows/main.yml

```diff
@@ -31,6 +31,29 @@ jobs:
         run: |
           cargo test --workspace --all-features
       - name: Run tests (without OGG)
+        run: |
+          cargo test --workspace --no-default-features --features="build"
+
+  tests-distribution-libflac:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install libraries
+        run: |
+          sudo apt-get install --yes --no-install-recommends libflac-dev libogg-dev
+      - name: Clone Git repository
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable${{ matrix.toolchain-suffix }}
+          override: true
+      - name: Run tests against distribution libraries
+        run: |
+          cargo test --workspace --no-default-features --features="ogg"
```

Comment:
I'm not sure whether the `ogg` feature makes a difference here?

Reply:
It doesn't, and we promise it doesn't. Let's make sure we keep that promise.

Reply:
But I don't understand what we are testing here.

The OGG test is disabled in this case, isn't it?

Reply:
That `ogg` really is ignored and doesn't try to link to libogg, which is not installed.

Reply:
This problem doesn't exist anymore.

Reply:
Which is why that line is no longer

Reply:
I know, I just wanted to acknowledge that, in case somebody else reads that in the future ...